### ISSUE TITLE
Document the commands

### ISSUE THREAD

**Issue by drmike** [lowendbox/lowendscript#5]
Just a suggestion to add a list of the commands to the readme file.

**Comment by xeoncross** [comment]
Good idea. The list of commands is in the script it's self though. Just call it with no options and it will explain itself to you.

**Comment by drmike** [comment]
It didn't the first time I tried but now it does.  Something's weird.

From looking at the original issue, there's some question about having to run the domain command before running wordpress command.  We didn;t have to prep first but now we do?  That's unclear.

**Comment by xeoncross** [comment]
The domain command is not to be run with the wordpress command. One is for setting up a raw PHP site for a domain - the other is for setting up a full wordpress site on a domain. If you run both of them on the same site you'll kill someone.
